src/main/java/org/jenkinsci/plugins/gitclient/JGitAPIImpl.java

```diff
@@ -1178,6 +1178,10 @@ void format(RevCommit commit, @Nullable RevCommit parent, PrintWriter pw, Boolea
         }
     }
 
+    public CleanCommand clean_() {
+    	throw new UnsupportedOperationException("No JGit support!");
```

Comment:
I had hoped it would be feasible to use the exclusion list as a way to generate the JGit inclusion list.  However, I can accept that isn't workable if constructing the that inclusion list is difficult or error-prone